plugins/core/qPCV/PCVCommand.cpp

```diff
@@ -0,0 +1,53 @@
+#include "PCVCommand.h"
+#include "PCV.h"
+#include "qPCV.h"
+
+//qCC_db
+#include <ccGenericMesh.h>
+#include <ccHObjectCaster.h>
+#include <ccPointCloud.h>
+#include <ccProgressDialog.h>
+
+static const char COMMAND_PCV[] = "PCV";
+
+PCVCommand::PCVCommand() :
+	Command("PCV", COMMAND_PCV)
+{
+}
+
+bool PCVCommand::process(ccCommandLineInterface& cmd)
+{
+	static int n_rays = 256;
```

Comment:
You didn't address this. Why are these `static`? As far as I can tell they should be `const` or `constexpr`.

Reply:
They should never have been static. They can't be const now that I've added reading options from the command line.